tests/modules/entity_module_bundle_plugin_test/src/Plugin/BundlePluginTest/BundlePluginTestInterface.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Drupal\entity_module_bundle_plugin_test\Plugin\BundlePluginTest;
+
+use Drupal\entity\BundlePlugin\BundlePluginInterface;
+
+interface BundlePluginTestInterface extends BundlePluginInterface {
```

Comment:
Added it as an example (cause you'd usually have other methods in there), but I agree that it contributes nothing in this particular case, I'll remove it.